README.md

```diff
@@ -1,6 +1,193 @@
 Job Restrictions Plugin for [Jenkins CI][1]
 =======================
-See [Job Restrictions Plugin Page][2] for more information.
+
+The plugin allows restricting job executions in order to change their
+behavior or to harden the security. With this plugin it is possible to
+configure nodes to accept only particular jobs. It is also possible to
+prevent job triggering by users and other jobs using various conditions.
+
+# About the plugin
+
+### Features:
+
+-   Execution restrictions on the node level
+    -   Example 1: Take jobs according to the specified name pattern
+        (e.g. allow running only "QA\_.\*" jobs on a node)
+    -   Example 2: Prevent execution of user jobs on the Jenkins master
```

Comment:
```suggestion
    -   Example 2: Prevent execution of user jobs on the Jenkins controller
```

The terminology has been changed recently

Reply:
Change made!